frontend/src/__tests__/cypress/cypress/tests/mocked/modelRegistry/modelVersion/modelVersions.cy.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable camelcase */
-import { mockDscStatus, mockK8sResourceList } from '~/__mocks__';
+import { mock403Error, mockDscStatus, mockK8sResourceList } from '~/__mocks__';
```

Comment:
i'm not seeing this mock403Error being used

Reply:
Agh - sorry, accidentally imported that